app/assets/stylesheets/application.css.scss

```diff
@@ -90,7 +90,15 @@ a, a:visited, a:link {
 }
 
 .flash {
-    text-align: center;
+  text-align: center;
+}
+
+.flash_notice {
+  font-size: 1.25em;
+  padding: 12px 0px 10px 0px ;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `12px 0px 10px` instead of `12px 0px 10px 0px`<br>Declaration should not have a space before the terminating semicolon<br>`0px` should be written without units as `0`